lightwood/api/predictor.py

```diff
@@ -65,11 +63,11 @@ def __init__(self, config=None, output=None, load_from_path=None):
         self.overall_certainty = None
 
     @staticmethod
-    def evaluate_mixer(mixer_class, mixer_params, from_data_ds, test_data_ds, dynamic_parameters,
+    def evaluate_mixer(config, mixer_class, mixer_params, from_data_ds, test_data_ds, dynamic_parameters,
```

Comment:
if we are passing the config, there is probably no need to pass the mixer class and mixer params part, right?

Reply:
There's no guarantee the mixer class will be in the config at the moment, and there could be mixer_params which we set by default but aren't in the config.

The mixer class gets assigned as: `mixer_class = NnMixer` by default and then we check to see if we check the config and override it if it's there.

Though I guess we could have a "default" mixer class in the config instead of setting the default in code, would be easier to understand for the user what the defaults are that way (and even for us).